NBA/update.py

```diff
@@ -0,0 +1,8 @@
+from nba_api.stats.endpoints import leaguegamefinder
+from datetime import date,timedelta
+
+class Update:
```

Comment:
Rename this class so it is more explicit.

What is it updating? How is it updating it?